chrome_manifest_v3/views/class_popupviewforms.js

```
@@ -0,0 +1,963 @@
+            chrome.runtime.sendMessage(hash, () => {
+              $status.html('Done');
+              setTimeout(() => {
+                $status.html('&nbsp;');
+              }, 2500);
+            });
+          } catch (error) {
+            this.parent.handleChromeAPIError(error, 'showMessage');
+          }
+          break;
+        case 'showsignout':
+          this.parent.showSignOutOptions();
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)